internal/strace/strace_test.go

```diff
@@ -109,6 +109,27 @@ func TestParseFileWritesToDifferentFiles(t *testing.T) {
 	}
 }
 
+func TestParseFileWriteWithZeroBytesWritten(t *testing.T) {
+	// Write calls where zero bytes are written are formatted as below where the write buffer argument does not include quotes.
+	input := "I1202 06:13:07.127115     312 strace.go:593] [  10:  10] npm init E write(0x2 host:[6], , 0x0)"
+
+	// We do not set WriteInfo if the bytes written is zero.
```

Comment:
Design point: could there be any benefit to logging zero-byte "writes"? As in, is there any distinction to be made between a write syscall to a file occurring with no bytes written vs no syscall at all?

Reply:
@oliverchang @calebbrown do either of you have any thoughts on this

Reply:
We should just try to provide comprehensive metadata (i.e. make this distinction in our data), and leave that decision to the consumer.

Reply:
Yeah I was really just asking whether we should record the writeinfo when zero bytes were written. Thanks!

Reply:
Makes sense. I'll add add this in.